presto-main/src/test/java/com/facebook/presto/sql/planner/assertions/ExpressionVerifier.java

```diff
@@ -243,6 +254,9 @@ else if (expression instanceof BooleanLiteral) {
         else if (expression instanceof DoubleLiteral) {
             return String.valueOf(((DoubleLiteral) expression).getValue());
         }
+        else if (expression instanceof DecimalLiteral) {
```

Comment:
Since we have this `getValueFromLiteral` , why do we need `visitDecimalLiteral`, `visitDoubleLiteral` .. above? Wouldn't `visitLiteral` be enough?
Or, rather, why isn't `getValueFromLiteral` method inlined?

Reply:
@findepi That's a good point. I don't see any reason to not inline `getValueFromLiteral`. Will do.

Reply:
@findepi @arhimondr I spoke too soon. `visitBooleanLiteral` and similar methods invoke `getValueFromLiteral` twice. Once, on `actual` value of known type and once again on `expected` value of unknown type. Hence, inlining wouldn't make sense here.

```
    @Override
    protected Boolean visitBooleanLiteral(BooleanLiteral actual, Node expected)
    {
        if (expected instanceof BooleanLiteral) {
            return getValueFromLiteral(actual).equals(getValueFromLiteral(expected));
        }
        return false;
    }
```